mmv1/products/gkebackup/BackupPlan.yaml

```diff
@@ -441,6 +458,13 @@ properties:
                   required: true
                   description: |
                     The name of a Kubernetes Resource.
+      - !ruby/object:Api::Type::Boolean
```

Comment:
Can you please add the update test for this field?
I recommend first setting this field to true, and then removing this field in the update test steps.

 The doc for the update test is [here](https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test)

Reply:
Thanks Zhenhua! Added. PTAL.